catroid/src/androidTest/java/org/catrobat/catroid/test/utils/IconsUtilTest.java

```diff
@@ -57,7 +57,7 @@ public void testAddIcons() {
 		assertNotNull("Control pos drawable not found.", correctDrawable);
 
 		Bitmap bitmap = ((BitmapDrawable) drawables[0]).getBitmap();
-		assertEquals("Icon size did not match the expected size.", IconsUtil.smallIconSizeBricks, drawables[0].getBounds());
+		assertEquals("Icon size did not match the expected size.", IconsUtil.getSmallIconSizeBricks(), drawables[0].getBounds());
```

Comment:
Well, that was just a fix for not compiling tests. But yeah you're right, normally this should be a separate PR :P